doc_source/cognito-authentication-extension.rst

```diff
@@ -41,7 +41,16 @@ The following examples require these :code:`using` statements:
 Use Basic Authentication
 ------------------------
 
-Create an :code:`AmazonCognitoIdentityProviderClient` :sdk-net-api:`AmazonCognitoIdentityProviderClient <TCognitoIdentityProviderCognitoIdentityProviderClient>` using :sdk-net-api:`AnonymousAWSCredentials <Runtime/TRuntimeAnonymousAWSCredentials>` which do not require signed requests. You do not need to supply a region, the underlying code will call :code:`FallbackRegionFactory.GetRegionEndpoint()` if a region is not provided. Create :code:`CognitoUserPool` and  :code:`CognitoUser` objects. Call the :code:`StartWithSrpAuthAsync` method with an :code:`InitiateSrpAuthRequest` that contains the user password.
+Create an
+:sdk-net-api:`<CognitoIdentityProvider/TCognitoIdentityProviderClient>`
```

Comment:
I think you still have to add text for the link before the url pattern.

Reply:
Ack, good catch. I'll fix.